content/en/blog/_posts/2021-XX-XX-writing-a-controller-for-pod-labels.md

```diff
@@ -0,0 +1,473 @@
+---
+layout: blog
+title: "Writing a Controller for Pod Labels"
+date: 2021-03-XX
+slug: writing-a-controller-for-pod-labels
+---
+
+**Authors**: Arthur Busser, Padok
+
+[Operators][what-is-an-operator] are proving to be an excellent solution to
+running stateful distributed applications in Kubernetes. Open Source tools like
+the [Operator SDK][operator-sdk] provide ways to build reliable and maintainable
+operators, and extending Kubernetes to implement custom scheduling needs is now
+easier than ever.
+
+Kubernetes operators run complex software inside your cluster. The open-source
+community has already built [many operators][operatorhub] for distributed
+applications like Prometheus, ElasticSearch, or ArgoCD. Even outside of
```

Comment:
```suggestion
Elasticsearch, or Argo CD
```

Reply:
The original seems OK @rolfedh ?

Reply:
Updated the suggestion to show the full line. The suggestion shows the correct capitalization and spacing of those two software names.

Reply:
Done 👍